scripts/travis/Jenkinsfile

```diff
@@ -9,7 +16,7 @@ def buildClosures(arg) {
             def(dir,testEnv) = testEnv_str.split(":")
             stage("$testEnv") {
```

Comment:
What are the potential values for $testEnv?

Reply:
The values are taken from the matrix section of the .travis.yml file.